data/core/rootview.lua

```diff
@@ -460,6 +483,7 @@ function RootView:update()
   self:update_drag_overlay()
   self:interpolate_drag_overlay(self.drag_overlay)
   self:interpolate_drag_overlay(self.drag_overlay_tab)
+  self.first_update = true
```

Comment:
Initialize it in `:new`, so we keep track of this.

Also a different name might be better, this sounds like it marks whether we're in the first update.